frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -125,6 +138,11 @@ class TensorboardViewer extends Viewer<TensorboardViewerProps, TensorboardViewer
                 className={classes(commonCss.buttonAction, css.button)}
                 disabled={this.state.busy}
                 color={'primary'}
+                title={
+                  this.state.tensorboardReady
+                    ? ''
+                    : 'Tensorboard is starting, and you may need to wait for a few minutes.'
```

Comment:
What does this look like? Can you share a demo?

Reply:
A tooltip like thing. https://drive.google.com/file/d/1t4A1jjUTa63whv97gZ2IXgp21GT2U_Dq/view

Reply:
Thanks for the demo!
I worry the tooltip shows too slowly, it might be hard to notice if someone just quickly clicks the button.

I think some simple text warning at the bottom (because the warning may show up and disappear, if it is shown on the top of the button, that will change the button's position on the screen) of the button works great too.

Reply:
Done